### DISCUSSION TITLE
CreateReport - response ID

### DISCUSSION THREAD

**Discussion by TomekKwiatkowski** [abuzuhri/Amazon-SP-API-CSharp#272] [General]
Hi,

First of all, thank for the great work.
Is there any chance to retrieve RequestID and Request Timestamp for Create Report ? We have an issues with requesting inventory  reports (Access to requested resource is denied. ) , other work perfect. I have checked setup on Amazon side and everything looks good, when asking Amazon support they require request ID and unfortunately, I'm unable to retrieve it for those request.

Best Regards
Tomek

**Comment by abuzuhri** [comment]
Yes it's available you can see *LastResponseHeader*
